api-reference/v1.0/resources/domain.md

```diff
@@ -0,0 +1,92 @@
+# domain resource type
+
+Represents a domain associated with the tenant.
+
+Use domain operations to associate domains to a tenant, verify domain ownership, and configure supported services.  Domain operations enable registrars to automate domain association for services such as Office 365. For example, as part of domain sign up, a registrar can enable a vanity domain for email, websites, authentication, etc.
+
+To associate a domain with a tenant:
+
+1. [Associate](../api/domain_post_domains.md) a domain with a tenant.
+
+2. [Verify](../api/domain_list_verificationdnsrecords.md) ownership of the domain. Verification records can be added to the domain through the domain registrar or DNS server configuration.
+
+3. [Indicate](../api/domain_update.md) the supported services you plan to use with the domain.
+
+4. [Configure](../api/domain_list_serviceconfigurationrecords.md) the domain for supported services. Configuration records can be added to the domain through the domain registrar or DNS server configuration.
```

Comment:
For this and #2 I'm not sure that it's clear to the reader that there is a step where you call Graph API to get information, and then you use that information to separately update your domain records through your registrar.  Can we make that clearer?